Python/Product/Cookiecutter/ViewModel/CookiecutterViewModel.cs

```diff
@@ -843,6 +843,9 @@ class CookiecutterViewModel : INotifyPropertyChanged {
                     CloningStatus = OperationStatus.Failed;
 
                     _outputWindow.WriteErrorLine(ex.Message);
+                    if (ex.GetType().Name == "FileNotFoundException") {
```

Comment:
No need to do this extra work. Just throw an exception and include the error string within it, like this: `throw new Exception(Strings.GitNotFoundError)`. Then like 845 will print the error message you need and you can get rid of this entire code block you added.